examples/metrics.py

```diff
@@ -0,0 +1,508 @@
+import json
+import os
+import re
+import statistics
+from collections import defaultdict
+from datetime import datetime, timedelta
+from typing import Dict, List
+
+import requests
+
+from examples.common.google_api import get_spreadsheets
+from examples.common.repos import ALL_REPOS
+
+ADMINS = {'childish-sambino', 'eshanholtz', 'thinkingserious'}
+DATE_TIME_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+
+ISSUE_TYPES = {'question', 'support', 'bug', 'enhancement', 'non-library', 'docs', 'security'}
+ISSUE_STATUSES = {'duplicate', 'invalid'}
+
+GOOGLE_SHEET_ID = '1cQOOT5aYxfXOSwEV0cJyf01KkV-uKCBJnKK3PHjouCE'
+
+
+def base_type():
+    return {'nodes': defaultdict(base_type), 'metrics': {}}
+
+
+class MetricCollector:
```

Comment:
Nice!

Thoughts on moving this to its own service (e.g. `metrics`) or add to the `github` service?

I think its beyond the scope of this work, but if you agree, let's make an issue.

Reply:
Meh, I don't see the need to move it to the service.